src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -296,7 +296,6 @@ private void ProcessProjectEvent(WorkspaceChangeEventArgs e, IAsyncToken asyncTo
                 switch (e.Kind)
                 {
                     case WorkspaceChangeKind.ProjectAdded:
-                        OnProjectAdded(e.NewSolution.GetProject(e.ProjectId));
```

Comment:
it might be simpler to pull out ths part of the change into its own PR that could be easily reviewed and submitted.  Right now it's a bit of a pain to have to go through it each time.